clients/src/main/java/org/apache/kafka/clients/consumer/internals/SubscriptionState.java

```diff
@@ -260,6 +270,10 @@ public void seek(TopicPartition tp, long offset) {
         return this.assignment.keySet();
     }
 
+    public Map<TopicPartition, TopicPartitionState> pendingUnsubscribePartitions() {
```

Comment:
Do we still need this method if we do not pass PendingUnsubscribe partitions to the rebalance listener?